deployments/with-creds/Makefile

```diff
@@ -30,6 +30,7 @@ diff-%: | ensure-creds-file-exists-% deps-%
 			--namespace=$* \
 			--detailed-exitcode \
 			--values=./.values.yaml \
+			--set=concourse.web.annotations.time=`date +%s` \
```

Comment:
Hey,

Does this mean that even when there are no changes (both in terms of secrets and other configs) we end up updating the pods?

Reply:
Yes we would, if you tried deploying with this change it would update all the pods. Why would one ever do a no-op change though?

I do think that this is a _clever_ workaround ([from here](https://github.com/helm/helm/issues/5218)) and makes it non-obvious what the behaviour is going to be. We actually couldn't find anywhere in the k8s docs (maybe we should've been looking at helm? unclear to me who's feature this is) that indicated that changing only annotations causes pods to be restarted.

We used this workaround because updating only the secrets resulted in just the secrets on the pods being remounted. Processes weren't being restarted, therefore changes weren't picked up :(

Reply:
We removed this from the Makefile and added it to the `values.yaml` and added a note to the README about this annotation. We also renamed the annotation to `rollingUpdate`. See new commit for full details.